doc/Native APIs.md

```diff
@@ -365,9 +393,19 @@ Each item in the `choices` array contains these fields:
 - `completed`: (boolean) Boolean indicating whether the state of the choice is
   correct.
 - `selected`: (boolean) `true` if the user selected this choice.
-- `tips`: (string) Tips formatted as a string of HTML.
+- `tips`: (array) A list of objects providing info about tips defined for the
+  problem. See below for more info.
```

Comment:
I don't think we need the `tips` in the `user_state` since they're already in the `student_view_data`. We should have one consistent place to find them, and since they're not user-specific, the `student_view_data` makes more sense.